cukedoctor-converter/pom.xml

```diff
@@ -54,14 +54,10 @@
         <dependency>
             <groupId>io.cucumber</groupId>
             <artifactId>cucumber-junit</artifactId>
-            <version>5.6.0</version>
-            <scope>test</scope>
```

Comment:
We do not need to specify version and scope here since this dependency is in a "DepdendencyManagement" section in the parent pom.xml